docs/source/api/apollo-gateway.mdx

```diff
@@ -135,6 +135,41 @@ You can specify any string value for the `name` field, which is used for identif
 **If you _aren't_ using managed federation,** either this field or `supergraphSdl` is required. Do not provide both.
 
 
+</td>
+</tr>
+
+<tr>
+<td>
+
+###### `uplinkEndpoints`
+
+`Array<string>`
+</td>
+<td>
+
+A list of Apollo Uplink URLs the gateway uses to poll for its managed configuration. For details and defaults, see [Apollo Uplink](/managed-federation/uplink/).
+
+Provide this field **only** if you are using managed federation.
```

Comment:
I think we should be extra clear here that even managed federation users probably don't need to configure this unless we explicitly say so.